src/cal/rad_to_refl.py

```diff
@@ -0,0 +1,201 @@
+                 'AUG13':1.01317,'AUG14':1.01299,'AUG15':1.01281,'AUG16':1.01263,
+                 'AUG17':1.01244,'AUG18':1.01225,'AUG19':1.01205,'AUG20':1.01186,
+                 'AUG21':1.01165,'AUG22':1.01145,'AUG23':1.01124,'AUG24':1.01103,
+                 'AUG25':1.01081,'AUG26':1.01060,'AUG27':1.01037,'AUG28':1.01015,
+                 'AUG29':1.00992,'AUG30':1.00969,'AUG31':1.00946,
+                 'SEP01':1.00922,'SEP02':1.00898,'SEP03':1.00874,'SEP04':1.00850,
+                 'SEP05':1.00825,'SEP06':1.00800,'SEP07':1.00775,'SEP08':1.00750,
+                 'SEP09':1.00724,'SEP10':1.00698,'SEP11':1.00672,'SEP12':1.00646,
+                 'SEP13':1.00620,'SEP14':1.00593,'SEP15':1.00566,'SEP16':1.00539,
+                 'SEP17':1.00512,'SEP18':1.00485,'SEP19':1.00457,'SEP20':1.00430,
+                 'SEP21':1.00402,'SEP22':1.00374,'SEP23':1.00346,'SEP24':1.00318,
+                 'SEP25':1.00290,'SEP26':1.00262,'SEP27':1.00234,'SEP28':1.00205,
+                 'SEP29':1.00177,'SEP30':1.00148,
+                 'OCT01':1.00119,'OCT02':1.00091,'OCT03':1.00062,'OCT04':1.00033,
+                 'OCT05':1.00005,'OCT06':0.99976,'OCT07':0.99947,'OCT08':0.99918,
+                 'OCT09':0.99890,'OCT10':0.99861,'OCT11':0.99832,'OCT12':0.99804,
+                 'OCT13':0.99775,'OCT14':0.99747,'OCT15':0.99718,'OCT16':0.99690,
+                 'OCT17':0.99662,'OCT18':0.99634,'OCT19':0.99605,'OCT20':0.99577,
+                 'OCT21':0.99550,'OCT22':0.99522,'OCT23':0.99494,'OCT24':0.99467,
+                 'OCT25':0.99440,'OCT26':0.99412,'OCT27':0.99385,'OCT28':0.99359,
+                 'OCT29':0.99332,'OCT30':0.99306,'OCT31':0.99279,
+                 'NOV01':0.99253,'NOV02':0.99228,'NOV03':0.99202,'NOV04':0.99177,
+                 'NOV05':0.99152,'NOV06':0.99127,'NOV07':0.99102,'NOV08':0.99078,
+                 'NOV09':0.99054,'NOV10':0.99030,'NOV11':0.99007,'NOV12':0.98983,
+                 'NOV13':0.98961,'NOV14':0.98938,'NOV15':0.98916,'NOV16':0.98894,
+                 'NOV17':0.98872,'NOV18':0.98851,'NOV19':0.98830,'NOV20':0.98809,
+                 'NOV21':0.98789,'NOV22':0.98769,'NOV23':0.98750,'NOV24':0.98731,
+                 'NOV25':0.98712,'NOV26':0.98694,'NOV27':0.98676,'NOV28':0.98658,
+                 'NOV29':0.98641,'NOV30':0.98624,
+                 'DEC01':0.98608,'DEC02':0.98592,'DEC03':0.98577,'DEC04':0.98562,
+                 'DEC05':0.98547,'DEC06':0.98533,'DEC07':0.98519,'DEC08':0.98506,
+                 'DEC09':0.98493,'DEC10':0.98481,'DEC11':0.98469,'DEC12':0.98457,
+                 'DEC13':0.98446,'DEC14':0.98436,'DEC15':0.98426,'DEC16':0.98416,
+                 'DEC17':0.98407,'DEC18':0.98399,'DEC19':0.98391,'DEC20':0.98383,
+                 'DEC21':0.98376,'DEC22':0.98370,'DEC23':0.98363,'DEC24':0.98358,
+                 'DEC25':0.98353,'DEC26':0.98348,'DEC27':0.98344,'DEC28':0.98340,
+                 'DEC29':0.98337,'DEC30':0.98335,'DEC31':0.98333,
+                 }
+
+#///////////
```

Comment:
This looks more like a script, rather than a program or a function. I would suggest that the following become part of a function which is then called from a main.

```python

def YourFunctionName():
    ...

if __name__=='__main__':
    YourFunctionName()
```